config/test.exs

```diff
@@ -5,3 +5,6 @@ config :goth,
 
 config :diplomat,
        token_module: Diplomat.TestToken
+
+config :ex_unit,
+       capture_log: true
```

Comment:
Is this still necessary? I updated bypass yesterday (as part of `v0.8.0`), and it seems to have gotten rid of the noise.